### DISCUSSION TITLE
Apakah bisa mencari Confidence Interval dengan melakukan permutation pada sample secara terpisah?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#22] [Q&A] [answered]
Sebelumnya, saya telah mencoba melakukan hipotesis testing dengan permutasi. Asumsi null hipotesis benar, maka distribusi statistik sample diacak dari gabungan total sample control dan sample eksperimen. Nah, apakah selanjutnya saya juga dapat mencari confidence interval dengan resampling permutasi namun kali dari ruang terpisah masing-masing sample control dan sample eksperimen?

**Comment by muhrisman** [comment, marked as answer]
Pada dasarnya, suatu confidence interval dari suatu dataset numerik akan selalu bisa didapatkan. Kalkulasi masing-masing confidence interval untuk setiap ruang contoh kejadian dapat dilakukan pada kasus bapak.

**Reply by Hertzspr** [reply]
![permutation buat confidence interval](https://user-images.githubusercontent.com/95233522/159688378-07d02c8b-5bac-4b4f-9cd0-92af56b126d3.JPG)

Saya coba selisih median tiap tiap hasil permutasi antar grup nilainya sama semua ternyata.

**Reply by Hertzspr** [reply]
![aaa](https://user-images.githubusercontent.com/95233522/159688885-986b3642-f1bc-4fcd-81eb-31aa91e54b0b.png)
